### PR TITLE
[documentation] Add hyperlinks to the actual directory in the README file.

### PR DESCRIPTION
**Pull request by AdarshRawat1**
## Changes 
 - Added directory link for examples directory in DPDK backend's Guide.
 - Added Directory link for "testdata/p4tc_samples" in Guide of "tc" Backend.
 - Added directory link for "examples" directory in "ubpf" backend's Guide.

### CODE REVIEW

**Review by jafingerhut — APPROVED** [review]
LGTM

**Review by fruffy — CHANGES_REQUESTED** [review]
Instead of making just the directory the link I would also include the term before. So `examples` and `testdata/p4tc_samples`

**Comment by AdarshRawat1** [comment]
> Instead of making just the directory the link I would also include the term before. So `examples` and `testdata/p4tc_samples`

Sure, that would be a better way of linking the resource.

File: backends/ubpf/README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ However, we introduced some modifications, which are listed below:
 
 ### How to use?
 
-The sample P4 programs are located in `examples/` directory. We have tested them with the [P4rt-OVS](https://github.com/Orange-OpenSource/p4rt-ovs) switch - 
+The sample P4 programs are located in [`examples/` directory](./examples). We have tested them with the [P4rt-OVS](https://github.com/Orange-OpenSource/p4rt-ovs) switch -
```
Comment (fruffy) [inline backends/ubpf/README.md:58]:
```suggestion
The sample P4 programs are located in the [`examples/` directory](./examples). We have tested them with the [P4rt-OVS](https://github.com/Orange-OpenSource/p4rt-ovs) switch - 
```

File: backends/tc/README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ The backend for TC reuses code from the p4c-ebpf for generating c file.
 
 ## How to use it?
 
-The sample p4 programs are located in the "testdata/p4tc_samples" directory.
+The sample p4 programs are located in the ["testdata/p4tc_samples" directory](../../testdata/p4tc_samples).
```
Comment (fruffy) [inline backends/tc/README.md:22]:
```suggestion
The sample p4 programs are located in the [`testdata/p4tc_samples` directory](../../testdata/p4tc_samples).
```

**Review by fruffy — APPROVED** [review]

**Pull request merged**